app/views/home/index.html.erb

```diff
@@ -10,7 +10,35 @@
     <% else %>
       <% if !@current_user.is_student %>
         <%= link_to (fa_icon 'graduation-cap', text: 'New Course'), new_course_path, class: 'btn btn-secondary btn-lg' %>
-        <br>
+
+        <p class="lead">My Lectures</p>
```

Comment:
"My Lectures" is missleading as it suggest the list shows all lectures of the lecturer, not just the running ones.

```suggestion
        <p class="lead">My Running Lectures</p>
```